### PR TITLE
Google Shopping Recommendations

### PR DESCRIPTION
**Pull request by Andy-Waine**
updated image placard css for google.com shopping recommendations to stop blacking out the image

BEFORE
GoogleAdBG-BEFORE

AFTER
GoogleAdBG-AFTER

### CODE REVIEW

**Comment by Andy-Waine** [comment]
@gusted please let me know if I can provide any additional info

**Comment by Gusted** [comment]
I'm not able to reproduce this problem.

image

**Comment by Gusted** [comment]
Could you check if any of the images(or their parent elements) has a `mix-blend-mode` property being set?

**Comment by Andy-Waine** [comment]
Hi @Gusted, I just checked - none of the placards have that property nor do their parent divs. The issue is still showing for me on two different computers with two different google accounts. Please let me know how you would like for me to proceed to resolve this issue.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -20105,6 +20105,14 @@ CSS
 .Gor6zc {
     background-color: white !important;
 }
+#platop0,
+#platop1,
+#platop2,
+#platop3,
+#platop4,
+#platop5 {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:20113]:
```suggestion
img[id^="platop"] {
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**